build/linux/Makefile

```diff
@@ -1,8 +1,10 @@
 TOP:=../..
+
 CXX?=clang++-3.9
-CXXFLAGS:=-std=c++1z -MMD -MP -g
+CXXFLAGS?=-std=c++1z -MMD -MP -g
 INCLUDES:=-I$(TOP)/external -I$(TOP)/include -I./
-ARFLAGS:=-rs
+AR?=llvm-ar-3.9
```

Comment:
Rather than rely on a specific version of AR for everybody, maybe just set that in your environment and just make AR?=ar here.